libfwupd/fwupd-self-test.c

```diff
@@ -95,6 +95,61 @@ fwupd_enums_func (void)
 }
 
 static void
+fwupd_remote_http_func (void)
+{
+	gboolean ret;
+	g_autoptr(FwupdRemote) remote = NULL;
+	g_autoptr(GError) error = NULL;
+
+	/* check fallback to HTTP */
+	remote = fwupd_remote_new ();
+	fwupd_remote_set_id (remote, "lvfs");
+	ret = fwupd_remote_load_from_data (remote,
```

Comment:
Why even bother hardcoding the data in the test suite.  Since it's the regular LVFS metadta I would think you can just load the keyfile in the test suite too.